### DISCUSSION TITLE
Break storybook when using package which using gatsby Link component

### DISCUSSION THREAD

**Discussion by lwyj123** [gatsbyjs/gatsby#32051] [Ideas / Feature Requests]
## Description
When using `gatsby-theme-i18n`, the storybook will breaks and show errors like below:

```
Module parse failed: Unexpected token (8:4)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
|   return (
>     <ul>
|       {config.map((locale) => (
|         <li key={locale.code}>
 @ /home/workspace/someproject/node_modules/gatsby-theme-i18n/index.js 5:0-59 5:0-59
```
After reading #5949 and https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/how-to/testing/visual-testing-with-storybook.md

I was found that it's because of the gatsby-theme-i18n using Gatsby's Link component and not be compiled.

in the https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/how-to/testing/visual-testing-with-storybook.md this can be fixed by using 
```js
config.module.rules[0].exclude = [/node_modules\/(?!(gatsby)\/)/]
```
but if some package like `gatsby-theme-i18n` is using the Link component, I have to change the exclude into 
```js
config.module.rules[0].exclude = [/node_modules\/(?!(gatsby|gatsby-theme-i18n)\/)/]
```
and I think those kind of problem should be fixed by someone or provide a better solution.

**Comment by LekoArts** [comment]
We'd be happy to review a PR adding a comment above the line to say that one should include other packages there, too, if they use Gatsby Link
